vsdehalo/denoise.py

```diff
@@ -70,15 +73,21 @@ def bidehalo(
 
     planes = normalise_planes(clip, planes)
 
+    if isinstance(bm3d_sigma, (int, float)):
+        bm3d_sigma = [float(bm3d_sigma)] * 2
+
     if matrix:
         clip = clip.std.SetFrameProp('_Matrix', int(matrix))
 
     process_chroma = 1 in planes or 2 in planes
 
-    if not cuda:
-        sigma_luma, sigma_chroma = 8, process_chroma and 6.4
-    else:
-        sigma_luma, sigma_chroma = 10, process_chroma and 8
+    if bm3d_sigma is None:
+        if not cuda:
+            sigma_luma, sigma_chroma = 5.0, process_chroma and 3.6
```

Comment:
Why were the default values made weaker?

Reply:
The current values nuke areas around lineart, especially in darker scenes.